src/client/App.js

```diff
@@ -38,6 +39,7 @@ function App() {
         <Route path="*" component={NotFoundPage} />
       </Switch>
       <SearchFilter/>
```

Comment:
<searchFilter> is visible in the Dashboard side bar. @pdsorensen @DiegoRuiz454  we need to remove it from here. Since this task is related to Milestone 3 we still don't have route to place it there.

![image](https://user-images.githubusercontent.com/19622435/108824696-511ea480-75c2-11eb-96cb-da69221e989e.png)